sniffles/sniffles.py

```diff
@@ -105,8 +106,22 @@ def start_generation(sconf):
     allrules = myrulelist.getParsedRules()
     # Retrieve Background Traffic percentage
     back_traffic_percent = sconf.getBackgroundTraffic()
-    back_dist_list = None
-    back_absent_proto = None
+    if sconf.getBackgroundTrafficRule() is not None:
+        bt_rule = BackgroundTrafficRule()
+        background_traffic_rule = sconf.getBackgroundTrafficRule()
```

Comment:
I think lines 111-119 can be deleted and the above line could/should be bt_rule = sconf.getBackgroundTrafficRule().